blocks/adaptiveform/adaptiveform.css

```diff
@@ -1,3397 +1,86 @@
-  }
-
-  .aem-Grid.aem-Grid--12>.aem-GridColumn.aem-GridColumn--offset--phone--8 {
-      margin-left: 66.6667%
-  }
-
-  .aem-Grid.aem-Grid--12>.aem-GridColumn.aem-GridColumn--offset--phone--9 {
-      margin-left: 75%
-  }
-
-  .aem-Grid.aem-Grid--12>.aem-GridColumn.aem-GridColumn--offset--phone--10 {
-      margin-left: 83.3333%
-  }
-
-  .aem-Grid.aem-Grid--12>.aem-GridColumn.aem-GridColumn--offset--phone--11 {
-      margin-left: 91.6667%
-  }
-
-  .aem-Grid.aem-Grid--12>.aem-GridColumn.aem-GridColumn--offset--phone--12 {
-      margin-left: 100%
-  }
-
-  .aem-Grid.aem-Grid--phone--12>.aem-GridColumn.aem-GridColumn--phone--1 {
-      float: left;
-      clear: none;
-      width: 8.33333%
-  }
-
-  .aem-Grid.aem-Grid--phone--12>.aem-GridColumn.aem-GridColumn--phone--2 {
-      float: left;
-      clear: none;
-      width: 16.6667%
-  }
-
-  .aem-Grid.aem-Grid--phone--12>.aem-GridColumn.aem-GridColumn--phone--3 {
-      float: left;
-      clear: none;
-      width: 25%
-  }
-
-  .aem-Grid.aem-Grid--phone--12>.aem-GridColumn.aem-GridColumn--phone--4 {
-      float: left;
-      clear: none;
-      width: 33.3333%
-  }
-
-  .aem-Grid.aem-Grid--phone--12>.aem-GridColumn.aem-GridColumn--phone--5 {
-      float: left;
-      clear: none;
-      width: 41.6667%
-  }
-
-  .aem-Grid.aem-Grid--phone--12>.aem-GridColumn.aem-GridColumn--phone--6 {
-      float: left;
-      clear: none;
-      width: 50%
-  }
-
-  .aem-Grid.aem-Grid--phone--12>.aem-GridColumn.aem-GridColumn--phone--7 {
-      float: left;
-      clear: none;
-      width: 58.3333%
-  }
-
-  .aem-Grid.aem-Grid--phone--12>.aem-GridColumn.aem-GridColumn--phone--8 {
-      float: left;
-      clear: none;
-      width: 66.6667%
-  }
-
-  .aem-Grid.aem-Grid--phone--12>.aem-GridColumn.aem-GridColumn--phone--9 {
-      float: left;
-      clear: none;
-      width: 75%
-  }
-
-  .aem-Grid.aem-Grid--phone--12>.aem-GridColumn.aem-GridColumn--phone--10 {
-      float: left;
-      clear: none;
-      width: 83.3333%
-  }
-
-  .aem-Grid.aem-Grid--phone--12>.aem-GridColumn.aem-GridColumn--phone--11 {
-      float: left;
-      clear: none;
-      width: 91.6667%
-  }
-
-  .aem-Grid.aem-Grid--phone--12>.aem-GridColumn.aem-GridColumn--phone--12 {
-      float: left;
-      clear: none;
-      width: 100%
-  }
-
-  .aem-Grid.aem-Grid--phone--12>.aem-GridColumn.aem-GridColumn--offset--phone--0 {
-      margin-left: 0%
-  }
-
-  .aem-Grid.aem-Grid--phone--12>.aem-GridColumn.aem-GridColumn--offset--phone--1 {
-      margin-left: 8.33333%
-  }
-
-  .aem-Grid.aem-Grid--phone--12>.aem-GridColumn.aem-GridColumn--offset--phone--2 {
-      margin-left: 16.6667%
-  }
-
-  .aem-Grid.aem-Grid--phone--12>.aem-GridColumn.aem-GridColumn--offset--phone--3 {
-      margin-left: 25%
-  }
-
-  .aem-Grid.aem-Grid--phone--12>.aem-GridColumn.aem-GridColumn--offset--phone--4 {
-      margin-left: 33.3333%
-  }
+/* stylelint-disable selector-class-pattern, selector-id-pattern, function-url-quotes */
```

Comment:
If all of the `aem-Grid` rules were deleted, is this still needed?

Reply:
yes, the linter also hates the different iterations of " .cmp-adaptiveform_widget" (class-pattern) and "#give-us--a-call' (id-pattern).